bundle/manifests/external-secrets-operator.clusterserviceversion.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
     categories: Security
     console.openshift.io/disable-operand-delete: "true"
     containerImage: ""
-    createdAt: "2025-05-14T12:04:01Z"
+    createdAt: "2025-05-20T08:13:26Z"
```

Comment:
If this field is changed upon every `make bundle` automatically we should make this a placeholder value or get rid of it. As in downstream build, Konflux render-templates be again tweaking this value!

Doesn't block us here, we can tackle this later.